numpy/ma/core.pyi

```diff
@@ -5,7 +5,7 @@ from collections.abc import Sequence
 from typing import Any, Literal, SupportsIndex, TypeAlias, TypeVar, overload
 
 from _typeshed import Incomplete
-from typing_extensions import TypeIs, deprecated
+from typing_extensions import Self, TypeIs, deprecated
```

Comment:
`Self` can also be imported from `typing` since Python 3.11

Reply:
woooah NumPy is now Python3.11+ 🤯  nice